server/ai_mediaserver.go

```diff
@@ -910,7 +923,9 @@ func (ls *LivepeerServer) CreateWhip(server *media.WHIPServer) http.Handler {
 					Type:        sourceTypeStr,
 					QueryParams: queryParams,
 					GatewayHost: ls.LivepeerNode.GatewayHost,
-				})
+					WhepURL:     whepURL,
+					UpdateURL:   generateGatewayLiveURL(ls.LivepeerNode.GatewayHost, streamName, "/status"),
+					StatusURL:   generateGatewayLiveURL(ls.LivepeerNode.GatewayHost, streamID, "/update")})
```

Comment:
Just now noticing that we have the stream ID for the update path but stream name in all the others ... I guess it's not new but seems strange

Reply:
Yeah it's annoying, I can't remember the reason now. I'll have a look to see if we could easily make the status endpoint take stream name too.